ceph.spec.in

```diff
@@ -775,6 +775,9 @@ Requires:       smartmontools
 Recommends:	smartmontools
 %endif
 %endif
+=======
```

Comment:
might want to resolve the conflict and repush.

Reply:
thanks @tchaikov , not sure how I missed that one